samples/package.json

```diff
@@ -8,10 +8,11 @@
     "node": ">=8"
   },
   "scripts": {
-    "test": "mocha system-test"
+    "test": "mocha system-test system-test/v3beta1/"
```

Comment:
We don't need to put the `v3beta1` tests in a different directory.  Let's just add another file in the `system-test` directory, and keep this simple :)

Reply:
I find the separation useful

```
.
samples/
samples/v1
samples/v1/translate_list_obj_v1.js
samples/v2beta1
samples/v2beta1/translate_list_obj_v2beta1.js
samples/system-test
samples/system-test/v1
samples/system-test/v1/translate_list_obj_v1.test.js
samples/system-test/v2beta1
samples/system-test/v2beta1/translate_list_obj_v2beta1.test.js
```

If I'm looking at samples or tests, my concerns are generally only about the version of the API I'm working with.

I would prefer working in this environment...

```
samples/system-test/v1/ { 10 files }.js
samples/system-test/v1beta1/ { 12 files }.js
samples/system-test/v1alpha1/ { 16 files }.js
```

More than I would like working in this environment...

```
samples/system-test/ { 38 files }.js
```

Especially as we move to a one-file-per-sample and one-file-per-test world.

`#misc` I intend to begin suffixing all region tags with the API version ~ but even with that, I prefer separation between the API versions.

Plus, GAPIC only understands 1 version of an API. When synth runs for an API with 3 versions, it triggers GAPIC gen's of samples and tests _for a specific version_. My current plan is `samples/VERSION` and `samples/system-test/VERSION`. It's best to have separate directories for this. If you'd like a different structure, that would be good to know!

Reply:
fwiw, as someone new to the libraries, having the separation by version folder did make it really easy for me to understand what belonged to what version. My brain liked the visual. Let me know which you decide though and I'll adjust if needed

Reply:
Cool - SGTM